tajo-docs/src/main/sphinx/index/how_to_use.rst

```diff
@@ -66,4 +67,10 @@ If you don't want to use the index scan feature anymore, you can simply disable
 
 .. note::
 
-     Once the index scan feature is enabled, Tajo currently always performs the index scan regardless of its efficiency. You should set this option when the expected number of retrieved tuples is sufficiently small.
+     Once index scan is enabled, Tajo currently always performs index scan regardless of its efficiency. You should set this option when the expected number of retrieved tuples is sufficiently small.
```

Comment:
I suggest:
``Once index scan is enabled, Tajo will perform 'index scan' if possible. In some cases, it may cause performance degradation. If you always want to get better performance, you should either enable or disable 'index scan' according to selectivity. Usually, the performance gain of index will increase when the selectivity is low.``